enhancements/update/cvo-log-level-api.md

```diff
@@ -0,0 +1,325 @@
+package. The field in the [`github.com/openshift/api/operator/v1`](https://github.com/openshift/api/tree/1f9525271dda5b7a3db735ca1713ad7dc1a4a0ac/operator/v1)
+package has been stable for several years. Thus, the addition may be directly
+released to GA. The implementation of the CVO to dynamically change its verbosity
+level is considered to be less complex.
+
+User facing documentation will be created in [openshift-docs](https://github.com/openshift/openshift-docs/).
+
+### Removing a deprecated feature
+
+No existing feature will be deprecated or removed.
+
+## Upgrade / Downgrade Strategy
+
+Not applicable.
+
+## Version Skew Strategy
+
+The relevant components for this enhancement are the CVO and the ClusterVersion
+CR. These components move closely together on updates and downgrades. During a
+version skew, the default log level will be used.
+
+A newer CVO that consumes an older ClusterVersion will receive an empty
+`OperatorLogLevel` field, and the CVO will continue using the default log level.
+
+An older CVO that consumes newer ClusterVersion will not notice the
+`OperatorLogLevel` field, and the CVO will continue using the default log level.
+
+## Operational Aspects of API Extensions
+
+This enhancement proposes a minor addition to the existing `ClusterVersion` CRD.
+This new addition will operationally impact only the CVO. It may increase or
+decrease its logs. Impacting the storage.
+
+## Support Procedures
+
+Not applicable.
+
+## Alternatives
+
+### Move the existing `LogLevel` type to a different package
+
+The proposed implementation does not follow the DRY (Don't repeat yourself) 
+principle, as it copies over the existing `LogLevel` data type from the
+[`github.com/openshift/api/operator/v1`](https://github.com/openshift/api/tree/master/operator/v1) package. This was proposed as the
+`ClusterVersionSpec` struct may not import the existing type, as this causes 
+import cycles.
+
+An alternative solution is to move the existing `LogLevel` type definition to a
+package that will not cause import cycles. It may be the
+[`github.com/openshift/api/config/v1`](https://github.com/openshift/api/tree/master/config/v1)
+package, different package, or a new package.
+This will not duplicate the code and may be a desired alternative. However, this
+would move a stable definition of a type corresponding to its package where it's
+heavily being used to another less logically corresponding package only due to
+an importing issue. Due to this issue, this alternative was not chosen.
+
+### The CVO state will be represented by a new Custom Resource
+
+The CVO will use a new Custom Resource (CR) for its state. Cluster operators 
+tend to have two associated CRs. `ClusterOperator` CR to configure a respective
+cluster operator state, and a CR that represents the operator's operand.
+
+The CVO is not a `ClusterOperator`; however, the CVO will try to adopt
+conceptually the structure of CRs used by `ClusterOperators`.
```

Comment:
I think you're confused here about the role of a ClusterOperator vs just operator group CRDs, see other comments

Reply:
Yep, I was. Thank you for the patience.